integrationtest/pg/pg_test.go

```diff
@@ -44,9 +54,60 @@ func TestCreateBucket(t *testing.T) {
 	require.Equal(t, 1, len(lst))
 }
 
+func TestArchiveTracker(t *testing.T) {
```

Comment:
The idea here is simple.

Spin things as usual, fire an archive, stop the Hub completely, re-spin up again, and continue to watch if the original fired archive is resumed and finalizes correctly.

This simulates the Hub being closed/crashing, and all being good after it starts again resuming things.